posthog/temporal/data_imports/external_data_job.py

```diff
@@ -152,45 +156,56 @@ async def run(self, inputs: ExternalDataWorkflowInputs):
             source_id=inputs.external_data_source_id,
         )
 
-        # TODO: split out the creation of the external data job model from schema getting to seperate out exception handling
         job_id, incremental = await workflow.execute_activity(
             create_external_data_job_model_activity,
             create_external_data_job_inputs,
-            start_to_close_timeout=dt.timedelta(minutes=5),
-            retry_policy=RetryPolicy(
-                initial_interval=dt.timedelta(seconds=10),
-                maximum_interval=dt.timedelta(seconds=60),
-                maximum_attempts=3,
-                non_retryable_error_types=["NotNullViolation", "IntegrityError", "BaseSSHTunnelForwarderError"],
-            ),
-        )
-
-        # Check billing limits
-        hit_billing_limit = await workflow.execute_activity(
-            check_billing_limits_activity,
-            CheckBillingLimitsActivityInputs(job_id=job_id, team_id=inputs.team_id),
             start_to_close_timeout=dt.timedelta(minutes=1),
             retry_policy=RetryPolicy(
                 initial_interval=dt.timedelta(seconds=10),
                 maximum_interval=dt.timedelta(seconds=60),
                 maximum_attempts=3,
```

Comment:
Probably should have this be `maximum_attempts=0` now that it's only hitting our infra?